test/server/data/expected-reports/minimal

```diff
@@ -4,18 +4,33 @@
   2/6 failed
 
   1) fixture1 - fixture1test2
-     Chrome - Error1: Lorem ipsum dolor sit amet, consectetur adipiscing elit
+     Chrome - Assertion failed at step Step:
+     eq(["12345678901"], ["00000000000"])
+
+     Arrays differ at index 0:
+
+     Expected: [0]: "12345678901"
+     Actual:   [0]: "00000000000"
+                     ^
 
   2) fixture1 - fixture1test2
-     Chrome - Error2: Ut enim ad minim veniam, quis nostrud exercitation
-     ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
-     dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat
-     nulla pariatur.
+     Chrome - Assertion failed at step Step:
+     notEq("test", "test")
```

Comment:
Can we align code by 4 spaces and separate it by new lines? IMHO it will look much better and clear